apps/desktop/src/key-management/biometrics/os-biometrics-mac.service.ts

```diff
@@ -21,26 +20,48 @@ export default class OsBiometricsServiceMac implements OsBiometricService {
     }
   }
 
-  async getBiometricKey(service: string, key: string): Promise<string | null> {
+  async isBiometricsAvailable(): Promise<boolean> {
+    return systemPreferences.canPromptTouchID();
+  }
+
+  async unlockWithBiometrics(userId: string, _clientkeyHalfB64: string): Promise<string | null> {
```

Comment:
🎨 typo and let's correct the types
```suggestion
  async unlockWithBiometrics(userId: string, _clientKeyHalfB64: string | undefined): Promise<string | null> {
```